### ISSUE TITLE
Usability. Voiceover not reading 'Find a site near you' as a link

### ISSUE THREAD

**Issue by helloantoine** [covid-alert-ny/covid-green-app#224]
Issue: In the page 'Learn how to keep others safe' , the link  'Find a site near you' in Get tested is not actionable
Solution: Make sure the element is detected as a link
![Image from iOS (5)](https://user-images.githubusercontent.com/16833643/91995135-58f63e00-ed2f-11ea-998f-7f0823fb74d1.png)

**Comment by floridemai** [comment]
@helloantoine  Can you check in the next build please. Thanks

**Comment by AlanSl** [comment]
@helloantoine iOS-specific issue done in https://github.com/project-vagabond/covid-green-app/pull/373 should be fixed in next build
